pgfinder/matching.py

```diff
@@ -47,53 +47,47 @@ def filtered_theo(ftrs_df: pd.DataFrame, theo_list: pd.DataFrame, user_ppm: int)
     matched_df = matching(ftrs_df, theo_list, user_ppm)
 
     # Create dataframe containing only theo_mwMonoisotopic & inferredStructure columsn from matched_df
-    filtered_df = matched_df.loc[:, "theo_mwMonoisotopic":"inferredStructure"]
+    filtered_df = matched_df.filter(["inferredStructure","theo_mwMonoisotopic"], axis=1)
```

Comment:
Good catch! This was some code I inherited! I tried applying your change, but I think that returns a view into the dataframe instead of a copy like `filter` and that led to some warnings. I've added `.copy()` to the end of your suggestion and that seems to work pretty well!